### ISSUE TITLE
Python 3 compatibility

### ISSUE THREAD

**Issue by Lesserkuma** [mvyskoc/avrtsb#1]
First of all, great project! As Python 2 has been discontinued, an updated version that works with Python 3 would be very much appreciated. Thanks!

**Comment by mvyskoc** [comment]
You are right. Now, I will have some time I try to look at it. I thought nobody use it so I did not have motivation make more updates.
